INDIVIDUAL_MEMBERSHIP.md

```diff
@@ -8,7 +8,7 @@ To engage as many community members as possible, the Individual Membership progr
 
 The Individual Membership program will roll out in two phases. 
 
-Phase 1: At launch, the program will only have a single tier. This will provide the OpenJS Foundation projects early validation of the program’s utility, give program developers the opportunity to receive feedback on tooling and workflow from members, and ease initial launch overhead for the implementers.
+Phase 1: At launch, the program will only have a single tier. This will provide the OpenJS Foundation, hereinafter referred to as the "Foundation", projects early validation of the program’s utility, give program developers the opportunity to receive feedback on tooling and workflow from members, and ease initial launch overhead for the implementers.
```

Comment:
Optional suggestion:

```suggestion
Phase 1: At launch, the program will only have a single tier. This will provide OpenJS Foundation (_the Foundation_) projects early validation of the program’s utility, give program developers the opportunity to receive feedback on tooling and workflow from members, and ease initial launch overhead for the implementers.
```

Reply:
Or even better, maybe get rid of "OpenJS Foundation" in that sentence and save it for later:

```suggestion
Phase 1: At launch, the program will only have a single tier. This will provide projects early validation of the program’s utility, give program developers the opportunity to receive feedback on tooling and workflow from members, and ease initial launch overhead for the implementers.
```